src/Tests/local.settings.json

```diff
@@ -1,8 +1,8 @@
 ﻿{
   "AzureWebJobsStorage": "UseDevelopmentStorage=true",
   "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-  //"NServiceBusConnectionString": "at-shared-ns.servicebus.windows.net",
-  "NServiceBusConnectionString": "UseLearningEndpoint",
+  "NServiceBusConnectionString": "das-at-shared-ns.servicebus.windows.net",
+  //"NServiceBusConnectionString": "UseLearningEndpoint",
   "NServiceBusLicense": "",
   "LearningTransportStorageDirectory": "C:\\git\\das\\das-funding-apprenticeship-earnings\\src\\.learningtransport",
 }
```

Comment:
local.settings.json should not be checked in

Reply:
Also I think .learningtransport should be included in the .gitignore - not really a specific issue to this PR but we should fix it